learning_table_tennis_from_scratch/hysr_one_ball.py

```diff
@@ -278,6 +289,8 @@ def __init__(self, hysr_config, reward_function):
         self._mujoco_ids = []
 
         # pam muscles configuration
+        if hysr_config.pam_config_file.startswith('~'):
+            hysr_config.pam_config_file = str(pathlib.Path.home())+hysr_config.pam_config_file[1:]
```

Comment:
There is `os.path.expanduser()` (or alternatively `pathlib.Path.expanduser()`) to replace `~` with path to home.

Reply:
this package still uses strings instead of pathlib.Path, so I would keep it this way, waiting for the refactor package to be done

Reply:
`os.path.expanduser()` works on strings, so

    hysr_config.pam_config_file = os.path.expanduser(hysr_config.pam_config_file)

should have the same effect as the current code.

Reply:
I was looking at the documentation of the pathlib exanduser method, which has to be applied on an instance of Path, so not helping in this case. Since I switched all operation from os.path to pathlib, I'd rather not bring os.path again.

Reply:
I see. I think using pathlib and converting to string in the end would also be fine in this case but if it is anyway going to be refactored soon, I'm good with keeping it as is.